content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -395,15 +395,7 @@ Here, `W` is the number of work items.
 
 ### Suspending a Job
 
-{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
-
-{{< note >}}
-Suspending Jobs is available in Kubernetes versions 1.21 and above. You must
-enable the `SuspendJob` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/)
-and the [controller manager](/docs/reference/command-line-tools-reference/kube-controller-manager/)
-in order to use this feature.
-{{< /note >}}
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
```

Comment:
People reading these docs might be running v1.21 or older. I recommend keeping a revised version of that text; maybe remind readers to check they're reading the docs for the version of Kubernetes they are running - you can link to https://kubernetes.io/docs/home/supported-doc-versions/ if that helps.